conf/app.routes

```diff
@@ -1,6 +1,5 @@
 # microservice specific routes
 
-->          /govuk-frontend                                                                    govuk.Routes
```

Comment:
this route is no longer needed based on the latest README of the lib:
https://github.com/hmrc/play-frontend-hmrc